addons/aircraft/CfgWeapons.hpp

```diff
@@ -16,8 +16,17 @@ class CfgWeapons {
         magazineReloadTime = 0.1;
     };
 
-    // bigger mag for comanche
     class CannonCore;
+    // Fix attrocious A-10 Cannon Dispersion
+    class Gatling_30mm_Plane_CAS_01_F: CannonCore {
+        class LowROF: Mode_FullAuto {
+            burst = 1;
+            reloadtime = 0.015;
```

Comment:
You need over 60 FPS to fire at this rate. This is impossible in multiplayer, the fire rate will vary depending on your FPS. This is not desirable.
Vanilla has 0.03, which is just about 34 FPS for constant fire rate.